backend/src/database/migrations/20220714064701-rename-users-column-name.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
Generally speaking, if a migration did not end up in the main branch, it can be edited in the pull request where it originated from. This is also valid and it can be a matter of preference and use case.

Reply:
Noted 👍